low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -64,13 +65,16 @@ class SSPLBundleGenerate(object):
         self.enclosure_wwn = "NA"
 
     def generate_sspl_support_bundle(self, parser):
-        if os.path.isfile(SSPL_SB_REQUEST):
-            msg = "One SSPL Support Bundle request is ongoing already, skipping current request..!"
+        allow_new_request = validate_sb_request()
```

Comment:
Need to update class name to SupportBundle and Generate() can be one of the interface here, renaming generate_sspl_support_bundle() to generate()

Reply:
Done